modules/oe_content_entity/modules/oe_content_entity_contact/oe_content_entity_contact.post_update.php

```diff
@@ -133,3 +133,18 @@ function oe_content_entity_contact_post_update_00007(): void {
     $field_storage_config->create($reference_field)->save();
   }
 }
+
+/**
+ * Set translatable "Social media" field.
+ */
+function oe_content_entity_contact_post_update_00008(): void {
```

Comment:
We cannot make these translatable on random sites that use this. They would have made them translatable already if they needed them. THis needs to be moved to EWCMS.